### PR TITLE
Add Tencent Kubernetes Engine starter workflow

### PR DESCRIPTION
**Pull request by mingshun**
Thank you for sending in this pull request. Please make sure you take a look at the [contributing file](https://github.com/actions/starter-workflows/blob/master/CONTRIBUTING.md). Here's a few things for you to consider in this pull request:

- [ ] Include a good description of the workflow.
- [ ] Links to the language or tool will be nice (unless its really obvious)

In the workflow and properties files:

- [x] Includes a matching `ci/properties/*.properties.json` file.
- [x] Use title case for the names of workflows and steps, for example "Run tests".
- [ ] The name of CI workflows should only be the name of the language or platform: for example "Go" (not "Go CI" or "Go Build")
- [ ] Include comments in the workflow for any parts that are not obvious or could use clarification.
- [x] CI workflows should run `push`.
- [ ] Packaging workflows should run on `release` with `types: [created]`.

Some general notes:

- [ ] Does not use an Action that isn't in the `actions` organization.
- [ ] Does not send data to any 3rd party service except for the purposes of installing dependencies.
- [ ] Does not use a paid service or product.

### CODE REVIEW

File: ci/properties/tencent.properties.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "name": "Build and Deploy to TKE",
+    "description": "Build a Docker container, publish it to TKE Registry, and deploy to TKE cluster.",
+    "iconName": "tencentcloud",
```
Comment (cdb) [inline ci/properties/tencent.properties.json:4]: Can you add an SVG icon to the `./icons` folder with this name please?

File: ci/properties/tencent.properties.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "name": "Build and Deploy to TKE",
```
Comment (cdb) [inline ci/properties/tencent.properties.json:2]: Can you change the name here (and in the workflow file too) to just be the platform, such as `TKE` or `Tencent Cloud` or `Tencent Kubernetes Engine`

File: ci/properties/tencent.properties.json (hunk 1 above)
Comment (mingshun) [reply ci/properties/tencent.properties.json:4]: done

File: ci/properties/tencent.properties.json (hunk 2 above)
Comment (mingshun) [reply ci/properties/tencent.properties.json]: done

**Pull request merged**